content/src/fieldConfig/config.json

```diff
@@ -49,6 +49,7 @@
   },
   "navigationDefaults": {
     "navBarUnnamedOwnedBusinessText": "Unnamed Business",
+    "navBarUnnamedForeignRemoteSellerWorkerText": "Unnamed Out-of-State Business",
```

Comment:
Is there a reason we're not using the `${OoS}` tag for this? Then all that logic is prebuilt, right?

Reply:
Actually... I guess that's because it's on for RS/RW right